src/templates/app/cashier/index.jsx

```diff
@@ -14,28 +14,81 @@ const IconWithLink = ({ button_id, button_link, img_id, img_src }) => (
     </div>
 );
 
-const DepositWithdrawButton = ({ is_payment_agent, is_virtual }) => (
-    <div className= {is_virtual ? 'gr-4 gr-12-m' : 'gr-2 gr-12-m'}>
+const DepositWithdrawButtonRealMoney = ({ is_virtual }) => (
+    <div className={is_virtual ? 'gr-4 gr-12-m' : 'gr-2 gr-12-m'}>
         <SeparatorLine className='gr-parent gr-hide gr-show-m gr-padding-10' invisible />
         <div className='gr-row gr-row-align-left gr-row-align-right-m'>
-            { is_virtual ?
+            {is_virtual ?
                 <div className='gr-adapt'>
                     <Button className='toggle button' id='VRT_topup_link' text={it.L('Reset my demo balance')} />
                 </div>
                 :
                 <React.Fragment>
-                    <div className='gr-adapt gr-no-gutter-m client_real invisible gr-parent'>
+                    <div className='gr-adapt gr-no-gutter-m client_real client_virtual invisible gr-parent'>
                         <Button
-                            className='toggle button client_real invisible deposit_btn_cashier'
-                            href={it.url_for(is_payment_agent ? '/cashier/payment_agent_listws' : '/cashier/forwardws?action=deposit')}
+                            className='toggle button client_real client_virtual invisible deposit_btn_cashier'
+                            id='fiat_deposit_link'
+                            href='javasript:;'
                             text={it.L('Deposit')}
                             text_className='deposit'
                         />
                     </div>
-                    <div className='gr-adapt client_real invisible'>
+                    <div className='gr-adapt client_real client_virtual invisible'>
                         <Button
-                            className='toggle button client_real invisible withdraw_btn_cashier'
-                            href={it.url_for(is_payment_agent ? '/paymentagent/withdrawws' : '/cashier/forwardws?action=withdraw')}
+                            id='fiat_withdraw_link'
+                            className='toggle button client_real client_virtual invisible withdraw_btn_cashier'
+                            href={it.url_for('/cashier/forwardws?action=withdraw')}
+                            text={it.L('Withdraw')}
+                            text_className='withdraw'
+                        />
+                    </div>
+                </React.Fragment>
+            }
+        </div>
+    </div>
+);
+
+const DepositWithdrawButtonCryptocurrencies = ({ is_cryptocurrencies_method }) => (
+    <div className='gr-2 gr-12-m'>
+        <SeparatorLine className='gr-parent gr-hide gr-show-m gr-padding-10' invisible />
+        <div className='gr-row gr-row-align-left gr-row-align-right-m'>
+            {is_cryptocurrencies_method ?
```

Comment:
Can we move this condition into the `id` and `href` of the buttons? I see that these segments are almost the same and they defer only at the `id` and `href` of the buttons. so maybe we can remove the redundant code.